file/src/main/java/akka/stream/alpakka/file/impl/FileTailSource.java

```diff
@@ -36,7 +36,8 @@
  *
  * <p>Aborting the stage can be done by combining with a [[akka.stream.KillSwitch]]
  *
- * <p>To use the stage from Scala see the factory methods in {@link akka.stream.alpakka.file.scaladsl.FileTailSource}
+ * <p>To use the stage from Scala see the factory methods in {@link
+ * akka.stream.alpakka.file.scaladsl.FileTailSource}
```

Comment:
It is not legal JavaDoc to break the `@link` tag. I recently change these lines to make Scaladoc more happy. Did the autoformat do this?

Reply:
Yes, it was auto-formatted. I have seen this flipping back and forth recently.

Reply:
And even if I manually add a linebreak before `{@link`, google java formatter reformats it like in this PR.

Reply:
Cool blend of Scaladoc markdown and Javadoc tag notation BTW. Guess what HTML it spits out...
https://doc.akka.io/api/alpakka/current/akka/stream/alpakka/file/impl/FileTailSource.html